hack/cleanup/do-janitor.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
can we make this under `hack/do-janitor/` directory? to make a bit organized directory
if you don't think so, just ignore this comment

Reply:
done @prksu